### PR TITLE
[MOS-647][MOS-671] BT volume control fixes

### PR DESCRIPTION
**Pull request by Lefucjusz**
Set of fixes for Bluetooth volume control issues:

* split OS volume from A2DP device volume;
* added translations for additional popup required
to indicate what volume is being set;
* fixed issue that on some devices setting the lowest
volume level would set the highest in reality;
* fixed Bluetooth auto-turnoff functionality;
* minor code cleanup.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

File: module-apps/apps-common/ApplicationCommonPopupBlueprints.cpp [hunk 1]
```diff
@@ -49,14 +49,15 @@ namespace app
                          std::to_string(volumeParams->getVolume()).c_str());
                 auto volume          = volumeParams->getVolume();
                 auto context         = volumeParams->getAudioContext();
+                auto source          = volumeParams->getRequestSource();
                 const auto popupName = resolveWindowName(gui::popup::ID::Volume);
                 if (const auto currentWindowName = getCurrentWindow()->getName(); currentWindowName == popupName) {
-                    updateCurrentWindow(std::make_unique<gui::VolumePopupData>(volume, context));
+                    updateCurrentWindow(std::make_unique<gui::VolumePopupData>(volume, context, source));
```
Comment (niepiekm) [inline module-apps/apps-common/ApplicationCommonPopupBlueprints.cpp:55]: Why not move the construction of the popup in front of the condition to keep the code DRY.

**Review by niepiekm — APPROVED** [review]

File: module-apps/apps-common/ApplicationCommonPopupBlueprints.cpp (hunk 1 above)
Comment (Lefucjusz) [reply module-apps/apps-common/ApplicationCommonPopupBlueprints.cpp]: Sure, fixed!

File: module-apps/apps-common/popups/VolumeWindow.cpp [hunk 2]
```diff
@@ -55,21 +55,41 @@ namespace gui
...
+
+        switch (source) {
+        case audio::VolumeChangeRequestSource::A2DP:
+            showBTAudioDeviceVolume();
+            return;
+        case audio::VolumeChangeRequestSource::HFP:
+            [[fallthrough]];
+        case audio::VolumeChangeRequestSource::HSP:
+            showCalling();
+            return;
```
Comment (dawidwojtas-mudita) [inline module-apps/apps-common/popups/VolumeWindow.cpp:88]: AFAIK `[[fallthrough]]` should be used if you do some operations in `case` and then you want to intentionally go to the next `case`. If `case` is empty you don't need to use `[[fallthrough]]`.

**Review by dawidwojtas-mudita — APPROVED** [review]

**Review by SP2FET — APPROVED** [review]

File: module-apps/apps-common/popups/VolumeWindow.cpp (hunk 2 above)
Comment (dawidwojtas-mudita) [reply module-apps/apps-common/popups/VolumeWindow.cpp:88]: Related link https://en.cppreference.com/w/cpp/language/attributes/fallthrough

**Pull request merged**